src/swarm/util/log/ClientStats.d

```diff
@@ -146,6 +152,40 @@ public class ClientStats : StatsLog
     }
 
 
+    /***************************************************************************
+
+        Construct the class to use the default StatsLog appender, registers it
+        with the applications' reopenable files extension (if present), and
```

Comment:
From swarm's point of view, there'll only be a single application, so I guess the apostrophe should be left-shifted once?